docker-compose.dep.yml

```diff
@@ -0,0 +1,140 @@
+# Spins up all the external dependencies to test the aries-controller
+# This requires varies HL Aries compoenets: indy pool, wallets db, multitenant agent, tails server; plus our own component the agency
+#
+version: '3.5'
+
+networks:
+    agency-network:
+      name: agency-network
+      driver: bridge
+      ipam:
+        driver: default
+        config:
+          -
+            subnet: 10.0.0.0/24
+
+services:
+    # External service dependencies for full test
+    protocol-indy-pool:
+        build:
+            context: resources
+            dockerfile: indy-pool-kiva.dockerfile
+            args:
+                pool_ip: '10.0.0.2'
+        image: protocol-indy-pool
+        container_name: protocol-indy-pool
+        working_dir: /home/indy
+        ports:
+            - "9701:9701"
+            - "9702:9702"
+            - "9703:9703"
+            - "9704:9704"
+            - "9705:9705"
+            - "9706:9706"
+            - "9707:9707"
+            - "9708:9708"
+        networks:
+            agency-network:
+                ipv4_address: 10.0.0.2
+
+    aries-guardianship-agency:
+        image: kivaprotocol/aries-guardianship-agency:0.0.37
```

Comment:
I have mixed feelings about this circular dependency.

Reply:
Ya me too, the problem is we need the agency in order to spin up agents to test against. Before the tests were all pretty simple (and didn't test much), so that you wouldn't know if something broke until it make it to one of the other repos. This at least allows us to be more confident about our aries-controller releases, but I agree I don't like having all these dependencies.